python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -452,6 +456,7 @@ function AIConfigEditorBase({
         id: promptId,
         modelName: newModel,
       });
+      logEventHandler?.("UPDATE_PROMPT_MODEL");
```

Comment:
Can we also include the model name for better insight into what models are being used?:
```
logEventHandler?.("UPDATE_PROMPT_MODEL", {model: newModel});
```